pkg/cloud/cloud.go

```diff
@@ -0,0 +1,25 @@
+package cloud
+
+import (
+	"strings"
+)
+
+const (
+	awsPlatformType = "aws"
+)
+
+// GetKubeletHostnameOverride returns correct hostname for kubelet if it should
+// be overridden, or an empty string otherwise.
+func GetKubeletHostnameOverride(platformType string) (string, error) {
+	var hostname string
+	var err error
+	switch strings.ToLower(platformType) {
+	case awsPlatformType:
```

Comment:
Done

Reply:
This is a bit strange. It makes it seem that the only supported platfrom is AWS. Should we think of `platformType` to be a required parameter? Internally it can be a no-op for certain platforms.

Reply:
@Fedosin
@aravindhp made a good point here, the default case should return no error, as we should be able to pass any supported cloud platform from WMCO without error. Sorry for the misdirection :pray:

Reply:
Ack! It's fixed now.